src/main/java/racinggame/domain/RacingGame.java

```diff
@@ -0,0 +1,68 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static racinggame.domain.CarFactory.createCars;
+import static racinggame.view.OutputView.getPositionDisplay;
+
+public class RacingGame {
+
+    private final List<Car> cars;
+    private final int tryCount;
```

Comment:
```suggestion
    private final TryCount tryCount;
```
이 값도 원시값인데 위와 같이 포장하면 의미가 있을까?